eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_manager.rb

```diff
@@ -42,6 +42,11 @@ def get_subscriber_exchange(channel, queue)
         return channel.fanout("#{_queue_name}.ex")
       end
 
+      def get_delay_exchange(channel, queue, delay)
+        _queue_name = EventQ.create_queue_name(queue.name)
+        channel.fanout("#{_queue_name}.#{delay}.d.ex")
+      end
+
```

Comment:
create method `get_queue_exchange` which should create an exchange for single usage by a queue.

exchange name: `"#{_queue_name}.ex"`